CondTools/RPC/data/RPCTwinMuxLinkMapInput.txt

```diff
@@ -0,0 +1,300 @@
+TM YB+1_S1 0 LB_RB+1_S1_BP1A 1580	CH0: W+1/RB1out/1 Backward 	CH1: W+1/RB2out/1 Backward 	CH2: W+1/RB1in/1 Backward 
```

Comment:
I agree, and our DPG confirmed these .dat files can be removed from _CondTools/RPC/data/_.  Should this be part of a new pull-request, or can I do it in this branch?